packages/utils/src/order/status.ts

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
q: shouldn't this be in types?

Reply:
unfortunately enums are transpiled to objects. and things that generate code we have to move to utils.

Reply:
the approach so far is having an enum in the utils and a type in the types

Reply:
Got it!